src/back/controllers/api/routes/challenge/getOne.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const challenge = require('../../models/challenge');
+
+const getOne = (req, res) => {
+    challenge.findOne({ _id: req.params.id }, (err, challengeDetails) => {
```

Comment:
There's [the special method](http://mongoosejs.com/docs/api.html#model_Model.findById) for this.